librz/core/cautocmpl.c

```diff
@@ -182,6 +182,18 @@ static void autocmplt_cmd_arg_macro(RzCore *core, RzLineNSCompletionResult *res,
 	}
 }
 
+static void autocmplt_cmd_arg_flag(RzCore *core, RzLineNSCompletionResult *res, const char *s, size_t len) {
+	RzFlagItem *item;
+	RzListIter *iter;
+	RzList *list = rz_flag_tags_list(core->flags, NULL);
```

Comment:
These are not `flag_tags`.

Use `rz_flag_all_list`. For now pass `false` to `by_space`... I guess we can split FLAGs later, if needed, in FLAG_INSPACE and FLAG.